docs/chart_best_practices/custom_resource_definitions.md

```diff
@@ -30,8 +30,8 @@ In this method, each chart must be installed separately.
 
 ### Method 2: Pre-install Hooks
```

Comment:
probably need to change the title here as well

Reply:
Good point, thanks for pointing that out

Reply:
Fix committed